include/circt/Dialect/FIRRTL/Translation/Import.h

```diff
@@ -1,11 +1,11 @@
-//===- FIRParser.h - .fir to FIRRTL dialect parser --------------*- C++ -*-===//
+//===- Import.h - .fir to FIRRTL dialect parser -----------------*- C++ -*-===//
```

Comment:
Please keep useful filenames so "open file" in an IDE works nicely.  Having everything named "Import.h" isn't super great for this.